validator/webui/@polymer/webui-statusbar/webui-statusbar.html

```diff
@@ -34,6 +41,10 @@
     <paper-material elevation="3">
       <div class="ampproject-result">Validation Status:
         <span class$="{{getClass(status)}}">{{status}}</span>
+        <p class$="{{getClass(status)}}">
+          Review our 'publishing checklist' to ensure successful AMP document distribution.
+          See <a href="https://bit.ly/2D54tM9">https://bit.ly/2D54tM9</a>
```

Comment:
maybe we can change the actual link text to something else as it's possible here or should we be consistent and display the bitly link as we do in other places.

Reply:
@pbakaus - Real quick sanity check, do we now have amp.dev short links? Or should we use bit.ly? 

I'm okay with `publishing checklist` linking out - or changing the link to `here`. 

Otherwise, LGTM :)

Reply:
we do have shortlinks now, but they need to be set up via a change to this file: https://github.com/ampproject/docs/blob/future/platform/config/go-links.yaml. Feel free to do so though if you think it would be useful here.

Reply:
done. thanks all. Will submit this once https://github.com/ampproject/docs/pull/2000 is submitted. Thanks